doc/tutorials/dnn/dnn_googlenet/dnn_googlenet.markdown

```diff
@@ -37,11 +37,9 @@ Explanation
 
 -# Read input image and convert to the blob, acceptable by GoogleNet
    @snippet dnn/caffe_googlenet.cpp Prepare blob
-   Firstly, we resize the image and change its channel sequence order.
+   Firstly, we resize the image and perform mean subtraction using cv::dnn::blobFromImage constructor
 
-   Now image is actually a 3-dimensional array with 224x224x3 shape.
-
-   Next, we convert the image to 4-dimensional blob (so-called batch) with 1x3x224x224 shape by using special cv::dnn::blobFromImages constructor.
+   The constructor then converts the image to a 4-dimensional blob (so-called batch) with 1x3x224x224 shape by using special cv::dnn::blobFromImages constructor.
```

Comment:
There is no `cv::dnn::blobFromImages` invocation here. So we can just say that we make 4-dimensional blob with size `1x3x224x224` applying all necessary pre-processing procedures like resizing and mean subtraction.